core/src/main/scala/kafka/Kafka.scala

```diff
@@ -17,28 +17,59 @@
 
 package kafka
 
-import java.util.Properties
-
 import joptsimple.OptionParser
+import kafka.s3shell.util.S3ShellPropUtil
 import kafka.server.{KafkaConfig, KafkaRaftServer, KafkaServer, Server}
 import kafka.utils.Implicits._
 import kafka.utils.{CommandLineUtils, Exit, Logging}
-import org.apache.kafka.common.utils.{Java, LoggingSignalHandler, OperatingSystem, Time, Utils}
+import org.apache.kafka.common.utils._
 
+import java.util.Properties
 import scala.jdk.CollectionConverters._
```

Comment:
It's better not to modify the original Kafka code to avoid conflicts as much as possible.